kasa/tests/fakeprotocol_smart.py

```diff
@@ -239,6 +241,10 @@ def _send_request(self, request_dict: dict):
         elif method == "set_dynamic_light_effect_rule_enable":
             self._set_light_effect(info, params)
             return {"error_code": 0}
+        elif method == "set_led_info":
+            info["get_led_info"]["led_status"] = params["led_rule"] != "never"
+            info["get_led_info"]["led_rule"] = params["led_rule"]
```

Comment:
Perhaps move into `_set_led` in a similar manner as it was done for light effects?

Reply:
Done